assets/.eslintrc.json

```diff
@@ -1,5 +1,8 @@
 {
   "env": {
     "browser": true
+  },
+  "rules": {
+    "strict": "off"
```

Comment:
Alright:
- keep `node: true`
- change to `strict: [2, "function"]`
- add `/* eslint-disable */` before GA

Sounds about right?